Build/Build.cs

```diff
@@ -240,12 +245,14 @@ class Build : NukeBuild
         });
 
     Target Pack => _ => _
-        .DependsOn(ApiChecks)
-        .DependsOn(TestFrameworks)
-        .DependsOn(UnitTests)
-        .DependsOn(CodeCoverage)
-        .DependsOn(CalculateNugetVersion)
+        .DependsOn(
```

Comment:
⛏️ It's a bit of taste thing, but I prefer the separate `DependsOn` lines.

Reply:
Correct :D

Mine would be that solution, but it's your project ;-)